src/Makefile

```diff
@@ -28,6 +31,12 @@ SHARED_CFLAGS += -fPIC -fvisibility=hidden -DSHARED
 CFLAGS ?= -g -O2 -Werror -Wall -std=gnu89
 ALL_CFLAGS += $(CFLAGS) -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 ALL_LDFLAGS += $(LDFLAGS)
+ifneq ($(EXTRA_CFLAGS),)
```

Comment:
in general having EXTRA_CFLAGS is probably useful on its own, but there is no need for `ifneq`, you can just use it above when appending to ALL_CFLAGS, same for EXTRA_LDFLAGS

Reply:
Yeah, I just keep the same style as https://github.com/libbpf/bpftool/blob/master/src/Makefile#L83

Reply:
I'd change it in bpftool's Makefile as well. Please don't add extra lines for something as simple as this. Please inline `$(EXTRA_CFLAGS)` and `$(EXTRA_LDFLAGS)` in ALL_CFLAGS and ALL_LDFLAGS definitions